values.yaml

```
@@ -77,6 +77,70 @@ global:
   # Enable security context constraints required for OpenShift
   sccEnabled: false
 
+  # Deploy auth sidecar to use openshift native features
+  authSidecar:
+    enabled: false
+    image: nginxinc/nginx-unprivileged
+    pullPolicy: IfNotPresent
+    default_nginx_settings: |
+      internal;
+      proxy_pass_request_body     off;
+      proxy_set_header            Content-Length          "";
+      proxy_set_header            X-Forwarded-Proto       "";
+      proxy_set_header            X-Original-URL          https://$http_host$request_uri;
+      proxy_set_header            X-Original-Method       $request_method;
+      proxy_set_header            X-Real-IP               $remote_addr;
+      proxy_set_header            X-Forwarded-For         $remote_addr;
+      proxy_set_header            X-Auth-Request-Redirect $request_uri;
+      proxy_buffering             off;
+      proxy_buffer_size           4k;
+      proxy_buffers               4 4k;
+      proxy_request_buffering     on;
+      proxy_http_version          1.1;
+      proxy_ssl_server_name       on;
+      proxy_pass_request_headers  on;
+      client_max_body_size        1024m;
+    default_nginx_settings_location: |
```

I think these defaults should be in the template, not here. The config here should be.
```suggestion
    nginx_settings_location: ~
```
Then the template should have `{{- if .Values.global.authSidecar.nginx_settings_location }}`